client/client_state.h

```diff
@@ -521,6 +519,24 @@ struct CLIENT_STATE {
 #endif
 
     KEYWORDS keywords;
+
+    double current_cpu_usage_limit() {
+        double x = global_prefs.cpu_usage_limit;
+        if (!user_active && global_prefs.niu_cpu_usage_limit>=0) {
+            x = global_prefs.niu_cpu_usage_limit;
+        }
+        if (x < 0.005 || x > 99) {
+            x = 100;
+        }
+        return x;
+    }
+    double current_suspend_cpu_usage() {
```

Comment:
Should this also be moved to the cpp file?

Reply:
same